res/values-tzl/strings.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
The automated job updating translations probably will fail if this file is removed. Please, let it there. Currently the build tools ignore unrecognized language folders instead of failing.